server/index.js

```diff
@@ -106,6 +106,44 @@ server.post('/api/players', async (req, res, next) => {
   next()
 })
 
+server.put('/api/players/:playerId', async (req, res, next) => {
+  if (!req.body) {
+    res.send(HttpStatus.BAD_REQUEST, 'Player IDs are missing.')
+    next()
+    return
+  }
+
+  const { playerId } = req.params
+  if (!playerId ) {
+    res.send(HttpStatus.BAD_REQUEST, 'Missing player ID.')
+    next()
+    return
+  }
+
+  const { playerName, slackName } = req.body
```

Comment:
What do you think about renaming these variables to `newPlayerName` and `newSlackName`?